CMakeLists.txt

```diff
@@ -123,10 +123,55 @@ endif()
 # Generate distribution packages with CPack.
 if(WIN32)
     set(CPACK_GENERATOR ZIP)
+elseif(LINUX)
+    set(CPACK_GENERATOR External)
+	set(CPACK_EXTERNAL_ENABLE_STAGING YES)
+	set(CPACK_EXTERNAL_PACKAGE_SCRIPT "${PROJECT_BINARY_DIR}/appimage-generate.cmake")
+
+	file(GENERATE
+	OUTPUT "${PROJECT_BINARY_DIR}/appimage-generate.cmake"
```

Comment:
I think we don't need to generate `appimage-generate.cmake` file here, just add it to `cmake` directory.